gendiff/file_operations.py

```diff
@@ -0,0 +1,28 @@
+import json
+from typing import Any, Dict
+
+import yaml
+from yaml.loader import SafeLoader
+
+
+def check_format(file_name: str) -> str:
```

Comment:
this function returns file format rather than just checking it, consider renaming